src/common/message/game-message.ts

```diff
@@ -38,6 +39,20 @@ export class MoveMessage extends Message {
         };
     }
 }
+export class SetChessMessage extends Message {
```

Comment:
can you add comments to these message declarations?

Reply:
of course colin